java/test/src/main/java/io/ray/test/NodeIpTest.java

```diff
@@ -0,0 +1,46 @@
+package io.ray.test;
+
+import io.ray.api.Ray;
+import org.apache.commons.lang3.SystemUtils;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@Test(groups = {"cluster"})
+public class NodeIpTest extends BaseTest {
+
+  private static final String NODE_IP = "127.0.0.2";
+
+  @BeforeClass
+  public void setUp() {
+    if (SystemUtils.IS_OS_MAC) {
```

Comment:
Hi @kfstorm, I don't have mac environments now. And I modified this to `IS_OS_LINUX` and works fine in the local test.

Reply:
That's alright. I just tried `SystemUtils.IS_OS_MAC` on my mac.